gdtf.py

```diff
@@ -616,57 +605,51 @@ def update_geometry(geometry):
         load_geometries(root_geometry)
         update_geometry(root_geometry)
 
-        # Add target for manipulating fixture
-        if add_target:
-            target = bpy.data.objects.new(name="Target", object_data=None)
-            collection.objects.link(target)
-            target.empty_display_size = 0.5
-            target.empty_display_type = "PLAIN_AXES"
-            target.location = (0, 0, -2)
-
-        def get_root():
-            for obj in objs.values():
-                if obj.get("geometry_root", False):
-                    return obj
-
         def get_axis(attribute):
+            axis_objects = []
             for obj in objs.values():
                 for channel in dmx_channels_flattened:
                     if attribute == channel["id"] and channel["geometry"] == obj.get("original_name", "None"):
-                        return obj
+                        obj["mobile_type"] = "head" if attribute == "Tilt" else "yoke"
+                        obj["geometry_type"] = "axis"
```

Comment:
The idea of "axis" in GDTF existed on the beginning, but similarly to the special naming of geometry, it is not used/useful anymore. Any geometry with Pan/Tilt needs to be treated as Axis. So looking for Axis is not useful. I learned the hard way, then a GDTF (pan/tilt) would work in gMA3 but not in BlenderDMX.

Reply:
@vanous I used this to collect the axis objects later in a list and to seperate heads and yokes for the constraints.
Did you managed to select a mode in the alienpix profile? This was not possible in my first test because no mode was found.

Reply:
@nrgsille76 that would happen if you copy pygdtf from the git version. The import for the utils package must be adjusted in the vendored version in the addon. For extension, it works with the default "import pygdtf", but addon needs the "from ... import pygdtf". Not sure why it would fail quietly. But for me this works now, from latest main:

![image](https://github.com/user-attachments/assets/6a44e7da-2478-4f9c-88e5-96493ecf637a)

Reply:
> I used this to collect the axis objects later in a list and to seperate heads and yokes for the constraints.

OK, yes. The development branch had some of this logic already. It is good to use in the main, it's just time consuming to re-implement it and i had been focusing on other stuff in development, while not working with too many multihead based fixtures.

Reply:
> > Did you managed to select a mode in the alienpix profile? This was not possible in my first test because no mode was found.
> 
> @nrgsille76 that would happen if you copy pygdtf from the git version. The import for the utils package must be adjusted in the vendored version in the addon. For extension, it works with the default "import pygdtf", but addon needs the "from ... import pygdtf". Not sure why it would fail quietly. But for me this works now, from latest main:
> 
> ![image](https://private-user-images.githubusercontent.com/3680926/354739310-6a44e7da-2478-4f9c-88e5-96493ecf637a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI3MDg5NTYsIm5iZiI6MTcyMjcwODY1NiwicGF0aCI6Ii8zNjgwOTI2LzM1NDczOTMxMC02YTQ0ZTdkYS0yNDc4LTRmOWMtODhlNS05NjQ5M2VjZjYzN2EucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgwMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MDNUMTgxMDU1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjNiY2JiMTMyMzRlZjg3ZTdjZDUxMGUyNzVlNjc0MjUwZGFlYWU3NTE2MTU1YjJkMWY4YTA0MTk4YWVjOTUwZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.5E4Bnn1e6MDlqg3WYW2LGYLqBuJAPbbl16ieMWdKgoI)

@vanous  I tested it with the latest extension version and got this traceback:

![DMX Extension bug](https://github.com/user-attachments/assets/b2be412f-c33c-4703-9509-ce4f59d62d30)

Reply:
You are on the extension, you need to use the addon (main) at this point. I will rebase extension branch before releasing.

Reply:
@nrgsille76 how do you sync data from repo to Blender?

Reply:
@vanous I used the official way, installed from extensions via preferences

Reply:
I see. What I do is that u use custom script path in Blender, and I git clone the repo there. This works well for the addon version, I can develop, switch branches, move back-forth and git is always the source of the truth.

For the extension testing, I either use another blender, or, I rename the custom path directory to cause blender not to see it and then I install the Extension either from the server or from local file, during testing.

This workflow works quite well for me and it's fast and reliable. What you must not do is to "Remove addon" from Blender preferences, ot would delete the git directory. (You can always clone).